src/core/Perspective.ts

```diff
@@ -420,12 +420,6 @@ export default class Perspective {
     }
 
     async getLinks(query: LinkQuery): Promise<LinkExpression[]> {
-        const remoteLinks = await this.callLinksAdapter('pull')
```

Comment:
Instead of removing it, is it better to trigger a `pull and save` call in background, but without waiting the result?